src/Components/SearchByInsurance/SearchByInsurance.js

```diff
@@ -8,16 +8,18 @@ const SearchByInsurance = ({ providerList, setFilteredProviderList }) => {
   const handleSearch = (event) => {
     event.preventDefault()
     const searchResults = filterProvider()
-    if (searchResults.length) {
+
+    if (searchResults?.length) {
```

Comment:
I'm not certain if the ? is required if we're using searchResults.length

Reply:
.length alone was throwing me an error in the console.  I added optional chaining after I was done digging through the docs about why.  I assume this is because it needs the trigger to account for undefined.